Android crash on startup vkquality class not found

on 1.13.0 some andoroid devices crash for me on startup. seems to be problematic devices trying to use vulkan. I tested with multiple devices on firebase testlab to confirm this is the issue.

vkquality class looks missing from my builds (extender build with cpp extensions, no special settings):

```
[...]
07-03 00:06:06.739 25920 12377 I GOS:StatusCollector: startCollecting is cancelled because of no user agreement.
07-03 00:06:06.742 12374 12374 I .app.soundalive: Using CollectorTypeCC GC.
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: VkQuality preflight unavailable (java.lang.ClassNotFoundException: com.google.android.games.vkquality.VKQuality), allowing Vulkan support probe.
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: java.lang.ClassNotFoundException: com.google.android.games.vkquality.VKQuality
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at java.lang.Class.classForName(Native Method)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at java.lang.Class.forName(Class.java:536)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at java.lang.Class.forName(Class.java:467)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at com.dynamo.android.DefoldVkQuality.runPreflight(DefoldVkQuality.java:72)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at com.dynamo.android.DefoldActivity.onCreate(DefoldActivity.java:253)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.Activity.performCreate(Activity.java:8953)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.Activity.performCreate(Activity.java:8922)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4146)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4322)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2685)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.os.Handler.dispatchMessage(Handler.java:106)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.os.Looper.loopOnce(Looper.java:230)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.os.Looper.loop(Looper.java:319)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at android.app.ActivityThread.main(ActivityThread.java:8906)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at java.lang.reflect.Method.invoke(Native Method)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:608)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: Caused by: java.lang.ClassNotFoundException: com.google.android.games.vkquality.VKQuality
07-03 00:06:06.752 12350 12350 W DefoldVkQuality: 	... 21 more
07-03 00:06:06.754 12374 12374 E .app.soundalive: Not starting debugger since process cannot load the jdwp agent.
[...]
07-03 00:06:07.475   832   832 I Zygote  : Process 12350 exited cleanly (1)
[...]
```

This was Galaxy A25 5G but reproducable with other low end devices

--- a/share/extender/build_input.yml
+++ b/share/extender/build_input.yml
@@ -439,7 +439,7 @@ platforms:
         ANDROID_SDK_VERSION:      "{{env.ANDROID_SDK_VERSION}}"
         ANDROID_BUILD_TOOLS_PATH: "{{env.ANDROID_SDK_BUILD_TOOLS_PATH}}"
     context:
-        engineJars: ["{{dynamo_home}}/ext/share/java/glfw_android.jar", "{{dynamo_home}}/share/java/gamesys_android.jar", "{{dynamo_home}}/share/java/sound_android.jar"]
+        engineJars: ["{{dynamo_home}}/ext/share/java/glfw_android.jar", "{{dynamo_home}}/ext/share/java/vkquality.jar", "{{dynamo_home}}/share/java/gamesys_android.jar", "{{dynamo_home}}/share/java/sound_android.jar"]
         engineLibs: ["engine", "engine_service", "dmbedtls", "zip", "profile", "profilerext", "profiler_remotery", "record_null", "gameobject", "ddf", "font", "resource", "gamesys", "gamesys_model", "gamesys_rig", "script_box2d_defold", "graphics_opengles", "graphics_vulkan", "graphics_transcoder_basisu", "basis_transcoder", "physics", "BulletDynamics", "BulletCollision", "platform", "LinearMath", "box2d_defold", "render", "render_font_default", "script", "luajit-5.1", "extension", "hid", "input", "particle", "rig", "dlib", "image", "dmglfw", "gui", "crashext", "sound", "tremolo", "liveupdate", "unwind", "decoder_wav", "decoder_ogg"]
         platformIncludes:   ["{{env.NDK_PATH}}/sources/android/native_app_glue", "{{env.NDK_PATH}}/sources/android/cpufeatures"]
         flags:      ["-fno-exceptions", "-fvisibility=hidden"]

